source/qcommon/common.c

```diff
@@ -230,6 +230,7 @@ void Com_Printf( const char *format, ... )
 	Q_vsnprintfz( msg, sizeof( msg ), format, argptr );
 	va_end( argptr );
 
+	printf("%s", msg);
```

Comment:
ah this probably wasn't supposed to get committed?.. although maybe we should have a flag for enabling console output ?